### PR TITLE
docs: release @excalidraw/excalidraw@0.11.0  🎉

### PR DESCRIPTION
**Pull request by ad1992**

### CODE REVIEW

File: src/packages/excalidraw/CHANGELOG.md [hunk 1]
```diff
@@ -89,6 +89,266 @@ Please add the latest change on the top under the correct section.
...
+
+- Support updating text properties by clicking on container [#4499](https://github.com/excalidraw/excalidraw/pull/4499)
+
+- Bind text to shapes when pressing enter and support sticky notes 🎉 [#4343](https://github.com/excalidraw/excalidraw/pull/4343)
+
+- Change `boundElementIds` → `boundElements` [#4404](https://github.com/excalidraw/excalidraw/pull/4404)
+
+- Support selecting multiple points when editing line [#4373](https://github.com/excalidraw/excalidraw/pull/4373)
+
+- Horizontally center toolbar menu
```
Comment (ad1992) [inline src/packages/excalidraw/CHANGELOG.md:136]:
```suggestion
- Horizontally center toolbar menu [commit link](https://github.com/excalidraw/excalidraw/commit/9b8ee3cacfec239617c357693cf2a3ca9972d2cb)
```

**Review by dwelle — APPROVED** [review]

File: src/packages/excalidraw/README.md [hunk 2]
```diff
@@ -347,6 +347,31 @@ To view the full example visit :point_down:
...
+}
+```
+
+Most notably, you can customize the primary colors, by overriding these variables:
+
+- `--color-primary`
+- `--color-primary-darker`
+- `--color-primary-darkest`
+- `--color-primary-light`
+- `--color-primary-chubb` — a slightly darker (in light mode), or lighter (in dark mode) `--color-primary` color to account for [Chubb illusion](https://en.wikipedia.org/wiki/Chubb_illusion).
```
Comment (dwelle) [inline src/packages/excalidraw/README.md:371]:
```suggestion
- `--color-primary-chubb` — a slightly darker (in light mode), or lighter (in dark mode) `--color-primary` color to fix contrast issues (see [Chubb illusion](https://en.wikipedia.org/wiki/Chubb_illusion)).
```

File: src/packages/excalidraw/CHANGELOG.md (hunk 1 above)
Comment (Gam3356) [reply src/packages/excalidraw/CHANGELOG.md]: Thank you guys! Willie Parish Jerome jr.

**Review by Gam3356 — COMMENTED** [review]
Thank you guys! Willie Please Jerome jr

**Pull request merged**